ChustaSoft.Tools.Authorization.AspNet/Controllers/AuthorizationController.cs

```diff
@@ -17,16 +19,21 @@ public class AuthorizationController : ApiControllerBase<AuthorizationController
         #region Fields
 
         private readonly ISessionService _sessionService;
+        private readonly SignInManager<User> signInManager;
```

Comment:
We can't directly use SignInManager here, ISessionService encapsulates Services that at same time are using Managers,

In that case should be at IUserService, and then by using it throug ISessionService

With this approach we can have a problem, we are forcing the implementation to use User instead of generic types